scripts/structure/gmx_densmap_hex_cross_sections.py

```diff
@@ -518,6 +518,7 @@ def finish_heatmap_plot(axis):
                 alpha=0.5)
         
         
+        r_prev = -1
```

Comment:
Is this a sensible default? Should be >= 0 because of `linspace` in `numpy_helper_functions::cross_section`.

Reply:
Actually, `r_prev` cannot (or not easily) be defined in advance and actually this is not necessary. After the first run of the loop, `r_prev` is set to the sample points along the first sample axis. And only from the second run of the loop onward it is compared with the current sample points. So, flake8 thinks it is not defined, but actually it is.

To circumvent this "error", we can set `r_prev` to any value in advance. I think the best choice would be `r_prev = None  # Avoid flake8: F821 undefined name 'r_prev'`.

Anyhow, I think I should remove this script from MDTools, because it's much too specific and no one will understand what it does. Even I'm not using it anymore and replaced it by the more generic `axial_hex_distribution_1nn.py` and `axial_hex_distribution_2nn.py` scripts.

Reply:
Done. Somehow forgot about `None` in Python.